bin/diagnostic.js

```diff
@@ -1 +1,14 @@
 'use strict';
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer >= 1 && answer <= 10) {
+   console.log("You guessed it!");
+   break;
+ } else {
+   console.log("the guess needs to be between 1 and 10");
+ }
+}
```

Comment:
Not sure if you didn't read the question entirely, or if you didn't know how to do the question. Here's one possible solution. Please let someone know if you are confused about this.
```
let answer = '';

while (answer !== '5') {
  if (answer < 1 || answer > 10) {
   console.log('Please try a number between 1 and 10');
 } else if (answer !== '') {
   console.log('That\'s not it!');
 }
  answer = ask("Guess my secret? ");
}
console.log("You guessed it!");
```